api/v1alpha1/loadbalancer_types.go

```diff
@@ -24,6 +27,15 @@ type LoadBalancer struct {
 	//
 	// +optional
 	ConsistentHash *ConsistentHash `json:"consistentHash,omitempty"`
+
+	// SlowStartWindow defines the duration of the warm up period for newly added host.
+	// If set, during slow start window, traffic sent to the newly added hosts will gradually increase in a linear manner.
+	// Currently this is only supported for RoundRobin and LeastRequest load balancers.
+	// For additional details,
+	// see https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/cluster.proto#config-cluster-v3-cluster-slowstartconfig
+	//
+	// +optional
+	SlowStartWindow *metav1.Duration `json:"slowStartWindow,omitempty"`
```

Comment:
should this live inside `roundRobin` and `leastRequest` since its not available for all lb types (like `consistentHash` ) ?

Reply:
or should we just rely on the CEL validation here ?

Reply:
IMO, CEL may be better, maybe the envoy extension supports other loadbalancer type. 👀

Reply:
Cool, no strong opinion here, current API is fine